### PR TITLE
updating vehicle_import_demo

### PR DESCRIPTION
**Pull request by robinsteuteville**
Updating vehicle_import_demo.py to utilize new demo testing.

### CODE REVIEW

File: python/fastsim/__init__.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 from . import fastsimrust
 from . import fastsimrust as fsr
 from . import parameters as params
-from . import utilities as utils
+from .utils import utilities as utils
```
Comment (kylecarow) [inline python/fastsim/__init__.py]: @robinsteuteville 
Try `from . import utils` here and in utilities/__init__.py add `from .utilities import *` and check out the effect :)

Comment (kylecarow) [reply python/fastsim/__init__.py:12]: ... and in utilities/__init__.py add `from .utilities import *` and check out the effect :)

Comment (kylecarow) [reply python/fastsim/__init__.py]: It's subtle, but this doesn't change anything about the package's namespace, and allows other utilities in the directory to be exposed as well (like if we wanted to expose the other file's functions under `fastsim.utils`, we could just edit utilities/__init__.py to import them as well)

Comment (robinsteuteville) [reply python/fastsim/__init__.py]: updated!

**Comment by kylecarow** [comment]
@calbaker This looks good to me!

**Comment by kylecarow** [comment]
@calbaker I think we reviewed this in the FASTSim dev meeting, is this good to merge?

**Review by calbaker — APPROVED** [review]

**Pull request merged**